Buildings/Media/Examples/BaseClasses/PartialProperties.mo

```diff
@@ -43,16 +44,23 @@ partial model PartialProperties
 protected
   constant Real conv(unit="1/s") = 1 "Conversion factor to satisfy unit check";
 
-  function checkState
+  function checkState "This function checks the absolute error in the state calculations"
     extends Modelica.Icons.Function;
     input Medium.ThermodynamicState state1 "Medium state";
     input Medium.ThermodynamicState state2 "Medium state";
+    input Real errAbs=errAbs "Absolute error threshold";
     input String message "Message for error reporting";
+  protected
```

Comment:
Are these "protected" intentionally repeated? They seem unnecessary to me.

Reply:
No, not intentional. Thanks for catching!

Reply:
Redundant `protected` removed.